tests/integration/regenerate.sh

```diff
@@ -59,6 +59,13 @@ for ((i = 0; i < num_of_tests; i++)); do
       sleep 1
       rm -rf logs
       rm -rf tests/integration/mock/$agent/$test_name/*
+      echo "echo -e \"/exit\n\" | SANDBOX_TYPE=$SANDBOX_TYPE WORKSPACE_BASE=$WORKSPACE_BASE \
```

Comment:
Or we could `set -v` to turn on verbose mode before the next command, and then `set +v` to turn it off again?

Reply:
Yeah I think `set +x` does the same--not sure on `set +v`

Reply:
Sorry yeah I meant by set +x

Reply:
Yeah I think `set -x` is better than `set -v`. The difference is, `-x` mode will replace the variable with its actual value, while `-v` mode will print out the script as it is.

Reply:
Thanks, fixed!